kafka-transport-binding.md

```diff
@@ -102,10 +100,10 @@ here.
 
 The receiver of the event can distinguish between the two content modes by
 inspecting the `ce_datacontenttype` [Header][Kafka-Message-Header] of the 
```

Comment:
shouldn't this be `content-type` ?
@clemensv

Reply:
@gunnarmorling if we fix this line would that address your confusion mentioned in #530 ?
I think the rest of the text in the spec is correct and doesn't need to be updated (except for the typo for `type`).

Reply:
Yes, using `content-type` consistently as the name of the header property helps 👍